### DISCUSSION TITLE
Moes BRT-100-TRV - no option to set time, weekday in Z2M and it is overwriting manual settings on device

### DISCUSSION THREAD

**Discussion by dt54171** [Koenkk/zigbee2mqtt#11103] [Devices]
hi! 
I am trying to use Programming_mode for Moes BRT-100-TRV. Z2M set it well. But in order to use this Programming_mode correct time and weekday should be set on device.
I haven't found a way how it can be done via Z2M integration.
There is an option to set time and weekday on device itself, but looks Z2M integration overwrites the values set to some default values.

One more question - if I am going to manage Moes BRT-100-TRV by HA via Z2M in manual mode - how to set target temperature for preset = manual?

**Comment by DomGrave** [comment]
Hi,
had the same issue with my BRT-100-TRV's.
Date/Time:
Install the chrony addon (ntp server) in your HA.

Temp:
Add thermostat card in your lovelace or set it with z2m directly
![grafik](https://user-images.githubusercontent.com/93572574/152648966-864b2fd5-6ed5-4242-b75f-3333cd4d8477.png)
![grafik](https://user-images.githubusercontent.com/93572574/152649020-b1a38f11-402b-446f-95fb-d1b7e58a758a.png)


regards
